README.rst

```diff
@@ -148,6 +148,9 @@ unicode_flag
   iOS and OS X. See https://en.wikipedia.org/wiki/Regional_Indicator_Symbol
   for details.
 
+code
+  The two letter country code for this country. (The code this country was instantiated with.)
```

Comment:
Well actually it's not necessarily the code the `Country` object was instanciated with. It could have been instanced with any type of ISO 3166-1 code (i.e. 3 character or numeric).

Let's just say:
> The two letter ISO 3166-1 code for this country.